gamma_inc_test.go

```diff
@@ -2,11 +2,13 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package mathext
+package mathext_test
```

Comment:
I seem to see that pattern more often than the other approach in the standard library, etc.